legend-engine-xt-relationalStore-executionPlan/src/main/java/org/finos/legend/engine/plan/execution/stores/relational/result/RelationalResult.java

```diff
@@ -495,6 +500,7 @@ public Object getTransformedValue(int columnIndex) throws SQLException
                 {
                     result = Boolean.valueOf(bool);
                 }
+                break;  // TODO: check this
```

Comment:
@PrateekGarg-gs @vikask-gs @gs-ssh16 
I believe break missing here is a bug and we should add this.
What are your views?

Reply:
yes

Reply:
yes